manifest.json

```diff
@@ -29,6 +29,15 @@
 					"en": "Choose a domain for Synapse",
 					"fr": "Choisissez un domaine pour Synapse"
 				},
+				"example": "matrix.domain.org"
+			},
+			{
+				"name": "server_name",
+				"type": "string",
+				"ask": {
+					"en": "Choose a name for Matrix accounts",
```

Comment:
Maybe can you specify that this is the domain name for the matrix accounts.

Reply:
Have a look at what I did here, it's more precise but maybe a too long explanation. Something in between would be good. https://github.com/YunoHost-Apps/synapse_ynh/compare/master...Thatoo:patch-1

I also had change the couple question/answer about public server that I personally found currently unclear.

Reply:
@Thatoo I'm fine with what you wrote

Reply:
I purpose to put something like that:
```suggestion
					"en": "Choose a domain name for your matrix user-ids. Note that the user-ids look like something like that : @user:domain.tld instead",
```

Reply:
Well, by starting with the word "choose", it sounds like we are completely free to choose whatever we want which is not the case. We have to remain in the main domain name zone of yunohost...
If yunohost is on _mydomain.tld_ and synapse is on _synapse.mydonain.tld_, the synapse server_name can be
_mydomain.tld_
or
_matrix.mydomain.tld_
but it can't be
_matrix.tld_ or _securechat.me_
...

Reply:
So waht do you think of this one ?
`If your synapse domain is a subdomain, you can select your main domain name to have your matrix user-ids looking like @user:domain.org instead of @user:synapse.domain.org. Otherwise enter the same subdomain as above.`